src/ui/buttons/button.jsx

```diff
@@ -39,12 +39,13 @@ type IndividualButtonProps = {|
     vault : boolean,
     merchantFundingSource : ?$Values<typeof FUNDING>,
     instrument : ?WalletInstrument,
-    inline? : boolean
+    experience? : string
 |};
 
 export function Button({ fundingSource, style, multiple, locale, env, fundingEligibility, i, nonce, flow, vault,
-    userIDToken, personalization, onClick = noop, content, tagline, commit, experiment, instrument, inline } : IndividualButtonProps) : ElementNode {
+    userIDToken, personalization, onClick = noop, content, tagline, commit, experiment, instrument, experience } : IndividualButtonProps) : ElementNode {
 
+    const inline = experience === EXPERIENCE.INLINE;
```

Comment:
It would be nice to make this variable more descriptive.
```suggestion
    const isInlineExperience = experience === EXPERIENCE.INLINE;
```

Reply:
Updating all suggestions with `inlineExperience`. Just think it reads better in an if-condition.